packages/providers/package.json

```diff
@@ -68,5 +68,8 @@
   "sideEffects": false,
   "tarballHash": "0x2582ad9a43f5113ef9143cf4f5b35cc5fe0cc709861834d5f5236ab06b2f4476",
   "types": "./lib/index.d.ts",
-  "version": "5.5.0"
+  "version": "5.5.0",
+  "devDependencies": {
```

Comment:
I think that this should be part of the `dependencies` and not `devDependencies`